src/system/RenderableSystem.lua

```diff
@@ -20,6 +20,27 @@ function RenderableSystem:setScene ( tag )
 
 end
 
+function RenderableSystem:update( dt )
```

Comment:
Cool: animation can be something we return to once we have units that 'do' stuff. We'll get the ticks into the game before widespread animations I think.